WordPress/Classes/Services/PostCoordinator.swift

```diff
@@ -243,6 +243,9 @@ class PostCoordinator: NSObject {
             if uploadedPost.isScheduled() {
                 self?.notifyNewPostScheduled()
             }
+            else if uploadedPost.isPublished() {
```

Comment:
[nit]

```suggestion
            } else if uploadedPost.isPublished() {
```

Reply:
Fixed in bfe27af8d36df3282b0247f5bb65d916edae9f7f